<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchCriteria</class>
 <widget class="QDialog" name="SearchCriteria">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>75</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>筛选条件</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="lblFirst">
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>过去</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spbFirstPeriod">
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="suffix">
       <string>日</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cbbQueryLogic">
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cbbComparisonField">
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cbbOperator">
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>322</x>
     <y>10</y>
     <width>211</width>
     <height>52</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="rbnPastAverage">
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>过去</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spbAveragePeriod">
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="suffix">
         <string>日平均</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="spbRelativePercentage">
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="suffix">
         <string>%</string>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
        <property name="value">
         <double>20.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QRadioButton" name="rbnAbsValue">
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>绝对值</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="spbAbsValue">
        <property name="font">
         <font>
          <family>Microsoft YaHei UI</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="suffix">
         <string/>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>99999.000000000000000</double>
        </property>
        <property name="value">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>161</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="btnConfirm">
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCancel">
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>rbnPastAverage</sender>
   <signal>clicked(bool)</signal>
   <receiver>spbAbsValue</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>447</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbnAbsValue</sender>
   <signal>clicked(bool)</signal>
   <receiver>spbAveragePeriod</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbnAbsValue</sender>
   <signal>clicked(bool)</signal>
   <receiver>spbRelativePercentage</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>469</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
